workshop/1 - Tooling and Deployment/README.md

```diff
@@ -0,0 +1,151 @@
+## 1 - Tooling and Deployment
+
+Welcome to the RWA .NET Workshop, we are going to learn how build reliable web applications using .NET and enterprise Cloud Apps. We are going to deploy a Ticket Store Application, Relecloud Concerts!
+
+For this guide, we assume you are using a Dev Container inside Visual Studio Code using Dev Containers.
+
+> For your convenience, we use Dev Containers with a fully-featured development environment. If you prefer to use Visual Studio, we recommend installing the necessary [dependencies](../../prerequisites.md) and skip to the deployment instructions starting in [Step 3](#3-log-in-to-azure).
```

Comment:
... we use [Dev Containers](link) to provide a complete development environment for the project.